src/liquid/tailwindcss-integration.md

```diff
@@ -0,0 +1,43 @@
+---
+eleventyNavigation:
+  key: Tailwind CSS Integration
+  parent: Liquid
+  order: 6
+layout: layout.njk
+title: Tailwind CSS Integration
+permalink: liquid/tailwindcss-integration/
+---
+
+
+# Tailwind CSS Integration
+
+Liquid boundles a [Tailwind CSS preset](https://tailwindcss.com/docs/presets) which you can use as a base for your own Tailwind CSS configuration:
+
+```js
+// tailwind.config.js
+const liquidPreset = require('@emdgroup-liquid/liquid/dist/css/tailwind-preset.js')
+
+module.exports = {
+  presets: [liquidPreset],
+}
+```
+
+## Disabling Tailwind's default configuration
+
+Liquid's Tailwind preset extends Tailwind's [default configuration](https://unpkg.com/browse/tailwindcss@%5E2/stubs/defaultConfig.stub.js) by default. If you want Liquid's preset to **not** extend Tailwind's default configuration, include an empty `presets` key in the preset: 
```

Comment:
This is also documented in Tailwinds docs. But if you want to explicitly mention it, I'd add that "not extending Tailwind" results in completely replacing the default config. This isn't clear by this paragraph.

Reply:
I added an additional note.